rust/schema/annotations.py

```diff
@@ -185,8 +185,15 @@ class _:
     ```
     """
 
+class FunctionOrMethodCallExpr(Expr):
```

Comment:
I think the name `FunctionOrMethodCallExpr` is a bit long. Perhaps shorted to `CallExprBase` ?

Reply:
I considered that, though I did not like breaking the `-Expr` postfixing. On the other hand I find `CallBaseExpr` confusing though. I also considered `AnyCallExpr`, which I might be persuaded to. I like the explicitness of `FunctionOrCallMehodExpr` that makes it clear what it is without having to go to its docs, but I do agree it's a bit long. Any thoughts on this @hvitved?

Reply:
Should ClosureExpr's also share the same base-class ?If so the name `FunctionOrMethodCallOrClosureExpr` would definitely get a bit too specific for my taste ;-)

Reply:
I think I prefer `CallExprBase`, despite breaking the suffix `Expr` invariant.

Reply:
ok, renaming now to `CallExprBase`. `ClosureExpr` is a function object (that may land or be referenced in `CallExpr::expr`), not a call, so it should not come beneath this class.

Reply:
Ah yes of course.